src/components/App/App.tsx

```diff
@@ -1,8 +1,60 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Route, Switch, NavLink } from 'react-router-dom';
 import { Header } from '../Header/Header';
+import mockPosts from '../../mockdata/mockPosts.json';
+import { Cards } from '../Cards/Cards';
 
-interface AppProps {}
+export interface Post {
+  type: string;
+  id: number;
+  attributes: {
+    user_id: number;
```

Comment:
Also we should look at the GraphQL types for IDs because they look weird to me. It looks like in some cases Graph has some sort of ID type and I don't know how that'll play with Typescript. It caused me some issues in testing because while up/downvotes came through as integers, the ids came through to me as strings (but specifically as GraphQL type !ID)

Reply:
Hmm okay, I guess I'll see what happens when I hook up to graphQL! I can change these types easily if they change with the BE data.